README.md

```diff
@@ -1 +1,179 @@
-# Technology-Innovation-Summit-2023
\ No newline at end of file
+# Technology Innovation Summit 2023
+
+Technology innovation Summit 2023
+
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <br/>
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Technology Innovation Summit 2023] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Technology Innovation Summit 2023]** is an online website for a technology conference with the big figures of the industry.
+
+> Video description: [@loomhandle](https://www.loom.com/share/216a834a707246da9fdccf8ffa452918)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack">HTML</a>
+
+### Tech Stack <a name="tech-stack">CSS</a>
+
+### Tech Stack <a name="tech-stack">JavaScript</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Innovation Technology Summit 2023]** Show informations about a technology conference that will take place soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [ ] Kindly record a five-minute video walking us through the project. The video should contain :
   - A description of the project.
   - A demo of the project features (different pages and different screen sizes).
   - You should also highlight some interesting piece of code or something you built that you are very proud of.